core/templates/dev/head/pages/about.html

```diff
@@ -426,6 +426,14 @@
   </script>
 {% endblock %}
 
+{% block footer_js %} 
+  {{ super() }}
+  <script>
+    {{ include_js_file('pages/About.js') }}
+    {{ include_js_file('domain/utilities/UrlInterpolationService.js') }}
```

Comment:
If this extends base.html do we need to include this js file again? Ditto elsewhere.